### ISSUE TITLE
Need to add SHA512 Class

### ISSUE THREAD

**Issue by darronschall** [mikechambers/as3corelib#20]
Originally filed by mikechambers on 2007-03-03T21:06:33

Need to add SHA512 Class

**Comment by darronschall** [comment]
Updated by mikechambers on 2008-07-02T16:27:17

Removed label Priority-Medium
Added label Priority-Low

**Comment by darronschall** [comment]
Updated by arcangel.nebula on 2008-08-14T07:36:18

Hello!

I have programmed the classes for SHA-512 encryption in Action Script 3.0. I want to
send to you the source code to consider if it´s possible to integrate with corelib´s
classes.

Also, i have programming SHA-384 encryption for Action Script 3.0 which I will could
to sent you shortly.

If you are interested, let me know and i´ll send you the source code.

E-mail:
juanmanuel[_at_]flashexperience[_dot_]net

Best regards!
Juanma
